app/population/views/edit/edit.njk

```diff
@@ -0,0 +1,40 @@
+{% extends "form-wizard.njk" %}
+
+{% macro display_field_group(group) %}
+  {% set fields = options.fieldGroups[group] %}
+
+  {% set key = "population::daily_view.labels." + group %}
+
+  <div class="govuk-grid-column-full govuk-!-margin-bottom-2">
+    <h2 class="govuk-heading-m">{{ t(key) }}</h2>
+
+    {% for fieldKey in fields %}
+      {% set field = options.fields[fieldKey] %}
+
+      {% if field.component and not field.skip %}
+        {{ callAsMacro(field.component)(field) }}
+      {% endif %}
+    {% endfor %}
+
+    {{ caller() if caller }}
+
+  </div>
+{% endmacro %}
+
+
+{% block fields %}
+
+  <div class="govuk-grid-row">
+    {% call display_field_group('total_space') %}
```

Comment:
If there's no data to display inside these you can call them without the `call` I believe:

```nunjucks
{{ display_field_group('total_space') }}
```

Reply:
Yeah, that's why I wrote the `if` block around the `content` - I clean this up in the subsequent PR, #1224 - I just wanted to be kinder to my future self and make it easier to keep track of what I was doing.